src/pages/settings/AboutPage/AboutPage.tsx

```diff
@@ -83,16 +89,19 @@ function AboutPage(props) {
                 action: waitForNavigate(Report.navigateToConciergeChat),
             },
         ];
-        return _.map(baseMenuItems, (item) => ({
-            key: item.translationKey,
-            title: translate(item.translationKey),
-            icon: item.icon,
-            iconRight: item.iconRight,
-            onPress: item.action,
+
+        return baseMenuItems.map(({translationKey, icon, iconRight, action, link}: MenuItem) => ({
+            key: translationKey,
+            title: translate(translationKey),
+            icon,
+            iconRight,
+            onPress: action,
             shouldShowRightIcon: true,
-            onSecondaryInteraction: !_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONST.CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor) : undefined,
+            onSecondaryInteraction: link
+                ? (event: GestureResponderEvent | MouseEvent) => ReportActionContextMenu.showContextMenu(CONST.CONTEXT_MENU_TYPES.LINK, event, link, popoverAnchor.current)
```

Comment:
`popoverAnchor` - Good catch! Though this was not noticeable as pageX, pageY always exist here:
https://github.com/Expensify/App/blob/c5ca9a810b529cdc8b8fceb179e9cfc01061355a/src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.tsx#L177-L182